triple-store/triple-store-impl-rdf4j/pom.xml

```diff
@@ -67,5 +67,28 @@
             <artifactId>powsybl-triple-store-api</artifactId>
             <version>${project.version}</version>
         </dependency>
-    </dependencies>
+
+		<!-- Test dependencies -->
+
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>log4j-over-slf4j</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-classic</artifactId>
+			<scope>test</scope>
+		</dependency>
+        <dependency>
+            <groupId>org.xmlunit</groupId>
+            <artifactId>xmlunit-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+	</dependencies>
```

Comment:
Fix the indentation issues and order the dependencies properly

Reply:
will sort test dependencies alphabetically by groupId

Reply:
It seems my remark about the coding style (indentation with spaces instead of tabs) has not been taken into account.